Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -13,11 +13,11 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 0,
-        "Patch": 6
+        "Minor": 1,
+        "Patch": 0
     },
     "demands": [],
-    "minimumAgentVersion": "2.0.0",
+    "minimumAgentVersion": "2.1.0",
```

Comment:
Because I updated the task version to 2.1.0 [here](https://github.com/Microsoft/vsts-tasks/pull/3869/files#diff-0bf716797e060f6c813008991524e17fL15) to follow [Semantic Versioning](http://semver.org/) because my change does "add functionality in a backwards-compatible manner".

Maybe I do not understand what minimumAgentVersion means, I thought it meant the minimum version number of the Task that the agent was allowed to use, the agent must have at least version 2.1 so that it includes the new logic that was added to azure-arm-compute.ts. If my understanding is incorrect can you please point me to the proper documentation for that field?